articles/qos_deadline_liveliness_lifespan.md

```diff
@@ -0,0 +1,241 @@
+In the case where a vendor chooses to continue operating with a reduced policy it must trigger a status event for the resource that can be handled by the new callback outlined above.
+This provides an application a way of being notified that even though their resource is operating, it is doing so with a reduced QoS.
+
+## FAQ
+
+- How does the Deadline policy take into account the additional overhead of ROS (such as deserialization) when determining if a deadline was missed?
+  - As a simplification it is not going to attempt to take into account any ROS overhead. A deadline will be considered missed if the rmw layer does not receive a message by the deadline and not if the user application on top of ROS does not receive it by the deadline. A new deadline policy could be added later that takes this into account.
+- What happens if multiple policy violations occur between the invocation of the callback? 
+  - This is dependent on the rmw implementation. If the rmw layer supports tracking multiple status events then it can return them in subsequent calls to rmw_take_status. If it does not support tracking multiple events at once then it may just return one of the events, such as the first or the last to occur in the sequence.
+- How do these QoS policies impact Actions?
+  - The existing Actions interface exposes that Actions are implemented with multiple Topics and it exposes the QoS settings for those topics. It is therefore up to the application implementing the action to specify the QoS policies for the different Topics. A new interface will need to be added to allow Actions to specify a ResourceStatusHandler, but that will be added in a later design review.
+- Why do Services not enforce deadline policies on connection like Publishers and Subscribers?
+  - This is a simplification being made for initial implementation. It could be changed later to enforce it, but that would require additional complexity in the existing DDS based implementations of the rmw to disallow connections based on non-DDS QoS settings.
+
+## Appendix A
+
+Definitions of the QoS policies from the DDS spec. 
+
+### Deadline
+
+This policy is useful for cases where a Topic is expected to have each instance updated periodically. On the publishing side this
+setting establishes a contract that the application must meet. On the subscribing side the setting establishes a minimum
+requirement for the remote publishers that are expected to supply the data values.
+
+When the Service ‘matches’ a DataWriter and a DataReader it checks whether the settings are compatible (i.e., offered
+deadline period<= requested deadline period) if they are not, the two entities are informed (via the listener or condition
+mechanism) of the incompatibility of the QoS settings and communication will not occur.
+
+Assuming that the reader and writer ends have compatible settings, the fulfillment of this contract is monitored by the Service
```

Comment:
```suggestion
Assuming that the `DataReader` and `DataWriter` ends have compatible QoS settings, the fulfillment of this contract is monitored by the Service
```

I think it would serve us well to be meticulous in our use of terminology, given that we have two similar systems.